source/assets/javascripts/locastyle/_tabs-button.js

```diff
@@ -5,34 +5,65 @@ locastyle.button = (function() {
 
   function init() {
     unbind();
-    bindClickOnTriggers();
+    addActivedButton();
+    bindClick();
     ariaTabs();
   }
 
   function unbind() {
-    $("[data-ls-module=button]").off("click.button");
+    $('[data-ls-module=button]').off('click.button');
   }
 
-  function bindClickOnTriggers() {
-    $("[data-ls-module=button]").on("click.button", function(evt) {
+  function bindClick() {
+    $('[data-ls-module="button"]').on('click.button', function(evt) {
       evt.preventDefault();
-      var $target = $($(this).attr("href") || $(this).data("target"));
+      var $target = $($(this).attr('href') || $(this).data('target'));
       var $buttons = '[data-ls-module=button]';
       deactivateElement(this, $target, $buttons);
       activateElement(this, $target);
+      removeChecked();
+      $(this).find('input').prop('checked', true);
     });
   }
 
+  function removeChecked(){
+    $('[data-ls-module="button"] input[type="radio"]').each(function(index, el){
+      $(el).removeAttr('checked');
+    });
+  }
+
+  function addChecked(el){
+    $(el).each(function(index, el){
+      $(el).attr('checked','checked');
+    });
+  }
+
+  function addActivedButton(){
+    $('.ls-tabs-btn-nav li.ls-active').each(function(){
+      if($(this).hasClass('ls-active') === true){
+
+        var $elem = $(this).find('[data-ls-module="button"]');
+        var $target = $(this).find('[data-ls-module="button"]').attr('href') || $(this).find('[data-ls-module="button"]').data('target');
```

Comment:
Use $elem to not having to duplicate "$(this).find('[data-ls-module="button"]')" at line 46.